dhis-2/dhis-services/dhis-service-analytics/src/test/java/org/hisp/dhis/analytics/event/data/EnrollmentAnalyticsManagerTest.java

```diff
@@ -257,7 +255,7 @@ void verifyWithProgramStageAndNumericDataElementAndFilter2()
             + programA.getUid() + ".pi = ax.pi and \"fWIAEtYVEGk\" is not null and ps = '"
             + programStage.getUid() + "' order by executiondate desc limit 1 )";
 
-        String expected = "ax.\"monthly\",ax.\"ou\"," + "coalesce(" + subSelect + ", 'NaN') as fWIAEtYVEGk" + "  from "
```

Comment:
I'm not sure but I think that returning 'NaN' was a requirement of some client months ago.

Reply:
see comment above